lib/praxis/action_definition.rb

```diff
@@ -31,7 +31,7 @@ def self.decorate_docs(&callback)
     end
 
     def initialize(name, resource_definition, **opts, &block)
-      @name = name
+      @name = name.to_sym
```

Comment:
Now that I think about it, do we do any checks in the resource definition before creating the action definition?